src/PopUp.js

```diff
@@ -0,0 +1,48 @@
+// This is what SelectedBeast would be if I had read the instructions fully.
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+// import Image from 'react-bootstrap/Image';
+// import data from './data.json';
+import InThePopUp from './InThePopUp.js';
+
+export default class PopUp extends Component {
```

Comment:
I would consider renaming this component as well. Modals and popups are different so perhaps a TA can help you rename this class. 👍